src/main/java/service/ReaderService.java

```diff
@@ -0,0 +1,7 @@
+package service;
+
+import java.util.List;
+
+public interface ReaderService {
```

Comment:
There is `Files.readAllLines()` in java core and I do not think we should reinvent the wheel creating our custom interface.